project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.5.5
\ No newline at end of file
+sbt.version=1.6.2
```

Comment:
Oops! Had to change it to get sbt to run but happy to remove from here or update to a more recent version.